### PR TITLE
feature: change default machine and filament from bambu to snapmaker

### PR DESCRIPTION
**Pull request by womendoushihaoyin**
change default machine and filament from bamboo to snapmaker

image

### CODE REVIEW

File: src/libslic3r/PresetBundle.cpp [hunk 1]
```diff
@@ -43,11 +43,17 @@ static std::vector<std::string> s_project_options {
...
 
-//BBS: add BBL as default
-const char *PresetBundle::BBL_BUNDLE = "Custom";
-const char *PresetBundle::BBL_DEFAULT_PRINTER_MODEL = "MyKlipper 0.4 nozzle";
-const char *PresetBundle::BBL_DEFAULT_PRINTER_VARIANT = "0.4";
-const char *PresetBundle::BBL_DEFAULT_FILAMENT = "My Generic PLA";
+////BBS: add BBL as default
+//const char *PresetBundle::BBL_BUNDLE = "Custom";
+//const char *PresetBundle::BBL_DEFAULT_PRINTER_MODEL = "MyKlipper 0.4 nozzle";
+//const char *PresetBundle::BBL_DEFAULT_PRINTER_VARIANT = "0.4";
```
Comment (meiliang1123) [inline src/libslic3r/PresetBundle.cpp:49]: We don't rename variables to keep minimum changes.

File: src/slic3r/GUI/ConfigWizard.cpp [hunk 2]
```diff
@@ -65,12 +65,12 @@ using Config::SnapshotDB;
 
 
 // Configuration data structures extensions needed for the wizard
-//BBS: set BBL as default
-bool Bundle::load(fs::path source_path, bool ais_in_resources, bool ais_bbl_bundle)
+// SM_FEATURE: set sm machine as default
```
Comment (meiliang1123) [inline src/slic3r/GUI/ConfigWizard.cpp:68]: keep bbl either

File: src/libslic3r/PresetBundle.cpp (hunk 1 above)
Comment (womendoushihaoyin) [reply src/libslic3r/PresetBundle.cpp:49]: Ok. I will delete them

File: src/slic3r/GUI/ConfigWizard.cpp (hunk 2 above)
Comment (womendoushihaoyin) [reply src/slic3r/GUI/ConfigWizard.cpp:68]: The two function has the same implement. The function depends the default machine you choose in code. So, the "Set BBL as default" version is meaningless in this context. In fact, "ais_default_bundle" is the more appropriate than the name "ais_bbl_bundle".

**Pull request merged**